examples/utils.py

```diff
@@ -80,9 +80,10 @@ def get_distributed_model(model_path):
         kill_tgis_container_if_exists()
 
     # TODO: Enforce validation that TGIS is mounting the same model type
-    _CONFIGURED_BACKENDS.clear()
     caikit.configure(
-        config_dict={"module_backends": {"priority": [TGISBackend.backend_type]}}
+        config_dict={
+            "module_backends": {"load_priority": [{"type": TGISBackend.backend_type}]}
```

Comment:
I wanted to try to run this but noticed these config changes and the `caikit.core.module_backends.module_backend_config` import appear to work with a `caikit` version prior to `0.10.0` (the current lower bound) - what version was this tested with? The caikit module backends appeared reorganized for `0.10.0` - https://github.com/caikit/caikit/pull/275

Reply:
this was tested on `0.9.1`. Need to get this updated.